lib/consyncful/item_mapper.rb

```diff
@@ -25,17 +25,26 @@ def id
       @item.id
     end
 
-    def mapped_fields(locale)
+    def mapped_fields(default_locale)
       fields = generic_fields
 
       @item.fields_with_locales.each do |field, value_with_locales|
-        value = value_with_locales[locale.to_sym]
-        next if value.is_a? Contentful::File # it is special
+        value_with_locales.each do |locale_code, value|
+          next if value.is_a? Contentful::File # assets are handeled below
 
-        assign_field(fields, field, value)
+          fieldname = locale_code == default_locale.to_sym ? field : "#{field}_#{locale_code.to_s.underscore}".to_sym
+
+          assign_field(fields, fieldname, value)
+        end
       end
 
-      fields[:file] = raw_file(locale) if type == 'asset'
+      if type == 'asset'
+        raw_files_by_locale.each do |locale_code, details|
+          fieldname = locale_code.to_sym == default_locale.to_sym ? :file : "file_#{locale_code.to_s.underscore}".to_sym
+
+          fields[fieldname] = details
```

Comment:
You can prevent the repetition on the `.to_sym` calls by moving it to line `:45`, where it uses the key.

```suggestion
          fieldname = locale_code == default_locale ? 'file' : "file_#{locale_code.to_s.underscore}"

          fields[fieldname.to_sym] = details
```

Reply:
Thanks again. That's much cleaner. Done!